vrk-kpa/suomidigi
# SUOPA-558 Exclude obsolete results in taxonomy view

To test: 
- `make fresh`
- Log in and flush all caches
- Go to f.e. https://suomidigi.docker.sh/asiakaslahtoisyys
  - Edit the taxonomy term and add these views: https://monosnap.com/file/BcEZDPL1JwdGChHPkh8PwxoelR9pVD
- Save the term and see that the taxonomy term view works just like before
